### ISSUE TITLE
Your meta description is missing! Add a compelling description of the page to attract more clicks. Aim for 100-130 characters.

### ISSUE THREAD

**Issue by eldhoabe** [eldhoabe/twoboyswebsite#6]
Your meta description is missing! Add a compelling description of the page to attract more clicks. Aim for 100-130 characters.
